### PR TITLE
island model created with island controller

### PR DESCRIPTION
**Pull request by aprositosakis**
The model and controller for the island and the corresponding views created!
Give feedback!

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
+  get 'islands/index'
+  get 'islands/show'
+  get 'islands/new'
+  get 'islands/create'
```
Comment (btoosey) [inline config/routes.rb]: Not necessary when we add resources :islands.

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
   devise_for :users
   root to: 'pages#home'
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :islands
```
Comment (btoosey) [inline config/routes.rb]: Perhaps change to:
resources :islands, only: [:index, :show, :new, :create]

File: config/routes.rb (hunk 1 above)
Comment (aprositosakis) [reply config/routes.rb]: All fixed with the unnecessary routes! Thanks for the comment.

**Comment by btoosey** [comment]
Looks good to me!

**Pull request merged**